modules/transaction-payment/src/lib.rs

```diff
@@ -64,6 +64,7 @@ pub use weights::WeightInfo;
 /// Fee multiplier.
 pub type Multiplier = FixedU128;
 
+pub type BalanceOf<T> = <<T as Config>::Currency as Currency<<T as frame_system::Config>::AccountId>>::Balance;
```

Comment:
isn't this the same as `PalletBalanceOf` below?

Reply:
Yeah don't know how I missed that :skull: